romancal/outlier_detection/outlier_detection_step.py

```diff
@@ -1,37 +1,202 @@
-"""
-Detect and flag outlier in a science image
-"""
+"""Public common step definition for OutlierDetection processing."""
 
-import roman_datamodels as rdm
+import contextlib
+import os
+from functools import partial
+
+from roman_datamodels import datamodels
+
+from romancal.datamodels import ModelContainer
+from romancal.outlier_detection import outlier_detection
 
 from ..stpipe import RomanStep
 
+# Categorize all supported versions of outlier_detection
+outlier_registry = {
+    "imaging": outlier_detection.OutlierDetection,
+}
+
```

Comment:
Is a registry needed if we have only one option? Just call the class.

Reply:
Refactored to call the class instead of using a registry.